code/game/gamemodes/changeling/powers/headcrab.dm

```diff
@@ -0,0 +1,15 @@
+/obj/effect/proc_holder/changeling/headcrab
+	name = "Last Resort"
+	desc = "We sacrifce our current body in a moment of need, stunning and damaging everyone nearby. If a dead body is nearby we infect it to raise again."
+	chemical_cost = 20
+	dna_cost = 1
+	req_human = 1
+
+/obj/effect/proc_holder/changeling/headcrab/sting_action(var/mob/user)
+	explosion(get_turf(user),0,0,2,0,silent=1)
+	spawn(5) // So it's not killed in explosion
+		var/mob/living/simple_animal/hostile/headcrab/crab = new(get_turf(user))
```

Comment:
There is a likelihood that the **user** is going to be destroyed (gibbed) after the spawn(5) starts executing. You should instead store the turf before the spawn and then use that to avoid a NullPointerException.